miner/miner.go

```diff
@@ -151,6 +151,7 @@ func (m *Miner) mine(ctx context.Context) {
 	var lastBase MiningBase
 
 	for {
+	minerStop:
```

Comment:
We could change this to
```suggestion
minerLoop:
	for {
```

And then use `continue minerLoop` instead of goto below

Reply:
Now it's changed